WordPress/Classes/ViewRelated/Stats/Period Stats/Overview/OverviewCell.swift

```diff
@@ -84,7 +84,7 @@ class OverviewCell: UITableViewCell, NibLoadable {
 
         setupFilterBar()
         updateLabels()
-        configureChartView()
+        configureChartViewIfNeeded()
```

Comment:
In a recent PR, we decided to remove the "if needed" prefix. I modified the behavior of the method so that it only installs the chart view once. This is more consistent with the notion of "if needed".